readthedocs/gold/models.py

```diff
@@ -52,10 +52,6 @@ class GoldUser(models.Model):
     stripe_id = models.CharField(max_length=255)
     subscribed = models.BooleanField(default=False)
 
-    # TODO: these are managed completely by Stripe now; we could remove them
-    last_4_card_digits = models.CharField(max_length=4, null=True, blank=True)
-    business_vat_id = models.CharField(max_length=128, null=True, blank=True)
```

Comment:
I assume we probably want to make sure that the users that did specify a VAT actually have this configured in Stripe on their subscription as well.

Reply:
Good point! I checked this with a small script:

```python
users = []
for g in GoldUser.objects.exclude(business_vat_id__isnull=True).exclude(business_vat_id=''):
    customer = stripe.Customer.retrieve(g.stripe_id)
    stripe_vat = customer.tax_ids.data[0].value
    if stripe_vat != g.business_vat_id:
        subscriptions = stripe.Subscription.list(customer=g.stripe_id)
        users.append((g.user.username, stripe_vat, g.business_vat_id, len(subscriptions.data)))
```

and I can say that all the users we have `business_vat_id` for has the same tax id on Stripe.